react-game/src/components/sounds/soundbacg.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react'
+import useSound from 'use-sound'
+
+import soundBG from './main-menu-2.mp3'
+
+const Soundbacgraund = () => {
+
+    const [volume, setVolume] = useState(0.3)
+    const [play, { stop, isPlaying }] = useSound(soundBG, {
+        volume,
+        loop: true})
+    
+    const playSound = () => {
+        if (isPlaying) {
+            return stop()
+        } else {
+            return play()
+        }
+    }
+
+    const plusVolume = () =>{
+        setVolume(volume + 0.1)
+    }
+
+    const minusVolume = () =>{
+        setVolume(volume - 0.1)
```

Comment:
Avoid magic numbers. Try to use constants for them, i.e. `const VOLUME_STEP = 0.1;`